src/mame/includes/playch10.h

```diff
@@ -72,22 +72,20 @@ class playch10_state : public driver_device
 	int m_IRQ_enable;
 	int m_pc10_bios;
 	tilemap_t *m_bg_tilemap;
-	DECLARE_WRITE8_MEMBER(up8w_w);
+	DECLARE_WRITE_LINE_MEMBER(up8w_w);
 	DECLARE_READ8_MEMBER(ram_8w_r);
 	DECLARE_WRITE8_MEMBER(ram_8w_w);
 	DECLARE_WRITE8_MEMBER(sprite_dma_w);
 	DECLARE_WRITE8_MEMBER(time_w);
-	DECLARE_WRITE8_MEMBER(pc10_SDCS_w);
-	DECLARE_WRITE8_MEMBER(pc10_CNTRLMASK_w);
-	DECLARE_WRITE8_MEMBER(pc10_DISPMASK_w);
-	DECLARE_WRITE8_MEMBER(pc10_SOUNDMASK_w);
-	DECLARE_WRITE8_MEMBER(pc10_NMIENABLE_w);
-	DECLARE_WRITE8_MEMBER(pc10_DOGDI_w);
-	DECLARE_WRITE8_MEMBER(pc10_GAMERES_w);
-	DECLARE_WRITE8_MEMBER(pc10_GAMESTOP_w);
-	DECLARE_WRITE8_MEMBER(pc10_PPURES_w);
+	DECLARE_WRITE_LINE_MEMBER(SDCS_w);
+	DECLARE_WRITE_LINE_MEMBER(CNTRLMASK_w);
+	DECLARE_WRITE_LINE_MEMBER(DISPMASK_w);
+	DECLARE_WRITE_LINE_MEMBER(SOUNDMASK_w);
+	DECLARE_WRITE_LINE_MEMBER(NMIENABLE_w);
+	DECLARE_WRITE_LINE_MEMBER(DOGDI_w);
+	DECLARE_WRITE_LINE_MEMBER(PPURES_w);
 	DECLARE_READ8_MEMBER(pc10_detectclr_r);
-	DECLARE_WRITE8_MEMBER(pc10_CARTSEL_w);
+	DECLARE_WRITE8_MEMBER(CARTSEL_w);
```

Comment:
CARTSEL might be straight from schematics or something? in that case I'd argue it was better documentation the way it is?